data/fields/lane_markings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "key": "lane:markings",
+    "type": "check",
+    "label": "Lane Markings"
```

Comment:
I think the more common term in the U.S. is “Lane Dividers”, and the official term is “lane line”, which should be a `term`. But “Lane Markings” is reasonably intuitive.

The wiki page makes it sound like this key can also indicate whether the road has a painted centerline. If so, “centerline”, “striped”, and “unstriped” should all be `terms`.